arviz/plots/densityplot.py

```diff
@@ -151,11 +151,14 @@ def plot_density(
 
         >>> az.plot_density([centered, non_centered], var_names=["mu"], bw=.9)
     """
-    if transform is not None:
-        data = transform(data)
+
     if not isinstance(data, (list, tuple)):
+        if transform is not None:
```

Comment:
Im being so pick here but could you move transform up to line 154 and delete one of the duplicate lines?

Reply:
But I don't think there is a duplicate, is there?
The data transformation is different depending on whether or not `data` is a list, so I'm not sure I can do the check and transform the data *before* checking `data`'s type 🤔